app/src/main/res/layout/fragment_rate_entry_detail.xml

```diff
@@ -0,0 +1,22 @@
+<?xml version="1.0" encoding="utf-8"?>
+<RelativeLayout
+    xmlns:android="http://schemas.android.com/apk/res/android"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent"
+    android:layout_margin="@dimen/activity_vertical_margin"
+    >
+
+    <ImageView
+        android:id="@+id/top_horizontal_entry_image_card"
+        android:layout_width="match_parent"
+        android:layout_height="@dimen/rating_image_card_height"
+        />
+
+    <android.support.v7.widget.RecyclerView
+        android:id="@+id/generic_recycler_view"
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content"
+        android:layout_below="@+id/top_horizontal_entry_image_card"
```

Comment:
Well... I'm not sure about best practices, but are you concerned with XML ordering? (which one gets created first)